packages/builder/src/components/common/bindings/BindingPanel.svelte

```diff
@@ -199,6 +200,7 @@
                   ]}
                   placeholder=""
                   height="100%"
+                  autofocus={autofocusEditor}
```

Comment:
You would need a similar entry for the JS editor around line 303.

Reply:
Good spot! Added.